blocks/core/ff_module/ff_module-button/ff_module-button.rt

```diff
@@ -1,15 +1,18 @@
-<button type="button"
+<button
+    type="button"
     title="{this.props.text}"
     id="{this.props.id}"
     disabled="{this.props.disabled}"
-    class="ff_module-button{this.props.modifier ? ' ff_module-button--'+this.props.modifier : ''}{this.props.classes ? ' '+this.props.classes : ''}{this.props.disabled ? ' ff_module-button--is-disabled' : ''}"
+    class="{this.generateClass('ff_module-button', this.props)}"
```

Comment:
Do you need to pass `this.props`? Presumably `generateClass()` already has access to them? Similarly, is there ever an occasion where it's called without `'ff_module-button'` as the first argument?